forgerock-openbanking-uk-aspsp-as/forgerock-openbanking-uk-aspsp-as-manual-onboarding/forgerock-openbanking-uk-aspsp-as-manual-onboarding-sample/pom.xml

```diff
@@ -151,7 +151,7 @@
 				<artifactId>dockerfile-maven-plugin</artifactId>
 				<configuration>
 					<skipPush>false</skipPush>
-					<repository>r.cfcr.io/openbanking/obri/register-sample</repository>
```

Comment:
Could use a property in the parent pom to make any future changes to the repository easier?

Reply:
I don't know about POMs but I don't see why not. For now I suggest merging this so we can restore stability to `master-dev` but I think this should be logged as an issue so we can revisit it later. I've logged https://github.com/ForgeCloud/ob-deploy/issues/471